android/anc/src/main/java/org/smartregister/fhircore/anc/ui/login/AncLoginService.kt

```diff
@@ -19,16 +19,49 @@ package org.smartregister.fhircore.anc.ui.login
 import android.content.Intent
 import javax.inject.Inject
 import org.smartregister.fhircore.anc.ui.family.register.FamilyRegisterActivity
+import org.smartregister.fhircore.anc.ui.otp.OtpLoginActivity
+import org.smartregister.fhircore.anc.ui.otp.OtpSetupActivity
 import org.smartregister.fhircore.engine.ui.login.LoginActivity
 import org.smartregister.fhircore.engine.ui.login.LoginService
+import org.smartregister.fhircore.engine.util.FORCE_LOGIN_VIA_USERNAME
+import org.smartregister.fhircore.engine.util.SharedPreferencesHelper
 
-class AncLoginService @Inject constructor() : LoginService {
+class AncLoginService @Inject constructor(val sharedPreferencesHelper: SharedPreferencesHelper) :
+  LoginService {
 
   override lateinit var loginActivity: LoginActivity
 
-  override fun navigateToHome() {
+  override fun navigateToHome(canSetOtp: Boolean) {
+    // Todo: check whether to setup PIN     or moveTo Register Home
+    if (canSetOtp) { // && sharedPreferencesHelper.read(OTP_PIN, "").isNullOrEmpty()) {
```

Comment:
what's the plan here?

Reply:
If I am not wrong this is to make the PIN login configurable. It will be available for all FHIR variants but only enable via the configs above.

Reply:
it is configurable, as App need to process PIN setup after login, so I have added enablePIN in login-view-configuration,
updated documentations for this methods and @param canSetPin